packages/datatrak-web/src/features/Tasks/TaskDetails/TaskDetails.tsx

```diff
@@ -142,83 +145,83 @@ export const TaskDetails = ({ task }: { task: SingleTaskResponse }) => {
   };
 
   return (
-    <Form onSubmit={handleSubmit(onSubmit)}>
+    <Wrapper>
       <LoadingContainer isLoading={isSaving} heading="Saving task" text="">
         <Container>
           <SideColumn>
-            <ItemWrapper>
-              <TaskMetadata task={task} />
-            </ItemWrapper>
-            <ItemWrapper>
-              <Controller
-                name="due_date"
-                control={control}
-                defaultValue={defaultValues.due_date}
-                render={({ value, onChange, ref }, { invalid }) => (
-                  <DueDatePicker
-                    value={value}
-                    onChange={onChange}
-                    inputRef={ref}
-                    label="Due date"
-                    disablePast
-                    fullWidth
-                    required
-                    invalid={invalid}
-                    disabled={!canEditFields}
-                  />
-                )}
-              />
-            </ItemWrapper>
-
-            <ItemWrapper>
-              <Controller
-                name="repeat_schedule"
-                control={control}
-                defaultValue={defaultValues.repeat_schedule}
-                render={({ value, onChange }) => (
-                  <RepeatScheduleInput
-                    value={value}
-                    onChange={onChange}
-                    disabled={!canEditFields}
-                    dueDate={dueDate}
-                  />
-                )}
-              />
-            </ItemWrapper>
-            <ItemWrapper>
-              <Controller
-                name="assignee_id"
-                control={control}
-                defaultValue={defaultValues.assignee_id}
-                render={({ value, onChange, ref }) => (
-                  <AssigneeInput
-                    value={value}
-                    onChange={onChange}
-                    inputRef={ref}
-                    countryCode={task.entity.countryCode}
-                    surveyCode={task.survey.code}
-                    disabled={!canEditFields}
-                  />
-                )}
-              />
-            </ItemWrapper>
+            <Form onSubmit={handleSubmit(onSubmit)}>
+              <ItemWrapper>
+                <TaskMetadata task={task} />
+              </ItemWrapper>
+              <ItemWrapper>
+                <Controller
+                  name="due_date"
+                  control={control}
+                  defaultValue={defaultValues.due_date}
```

Comment:
I don't think that these default values are needed if the default value is set at the form level above